v8py/convert.cpp

```diff
@@ -201,9 +201,8 @@ Local<Value> js_from_py(PyObject *value, Local<Context> context) {
     }
 
     if (PyList_Check(value) || PyTuple_Check(value)) {
-        int length = PySequence_Length(value);
         Local<Array> array = Array::New(isolate, length);
```

Comment:
Hey @tbodt while building and testing this out, I noticed that  length was not replaced with PySequence_Length(value) , seemed too small to open a PR for.

Reply:
fuck